lib/modules/commentNavigator.js

```diff
@@ -222,6 +225,9 @@ export function getPostsByCategory() {
 						.join(', ');
 					_posts[category] = highlightedUserSelector ? document.querySelectorAll(highlightedUserSelector) : [];
 					break;
+				case 'tagged':
+					_posts[category] = document.querySelectorAll('.noncollapsed .userTagLink.hasTag');
```

Comment:
This will also select user tags within comments (i.e. mentioned usernames), not just authors.

Probably need to use: `.noncollapsed .tagline .userTagLink.hasTag`